components/datetime/src/any/datetime.rs

```diff
@@ -102,10 +102,10 @@ use writeable::Writeable;
 ///     .expect("Failed to create DateTimeFormatter instance.");
 ///
 /// let manual_value = dtf.format(&manual_datetime).expect("Calendars should match");
-/// assert_writeable_eq!(manual_value, "Mar 28, 15 Heisei, 12:33\u{202f}PM");
+/// assert_writeable_eq!(manual_value, "Mar 28, 15 Heisei, 12:33 PM");
```

Comment:
Nit (can be follow-up): It was suggested that we leave an unrendered comment (`# //`) stating that there is a NNBSP on the following line to help with refactoring

Reply:
@echeran Did this ever get done? Looks like no